CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 NativeScript CLI Changelog
 ================
 
+3.4.0 (2017, December 20)
+==
+
+### Fixed
+* [Fixed #3268](https://github.com/NativeScript/nativescript-cli/issues/3268): tns run android - EMFILE: too many open files
+* [Fixed #3202](https://github.com/NativeScript/nativescript-cli/issues/3202): `tns update <version>` does not work
+* [Fixed #3187](https://github.com/NativeScript/nativescript-cli/issues/3187): Android tns debug, crashing when there is a response from server
+* [Fixed #2968](https://github.com/NativeScript/nativescript-cli/issues/2968): Separate prepare of JavaScript from preparation of native code
```

Comment:
This is not a fix, this is implementation. However I believe we do not need it in the release notes as this is just technical implementation.